airbyte-integrations/connectors/destination-redis/src/main/java/io/airbyte/integrations/destination/redis/RedisDestination.java

```diff
@@ -35,21 +36,16 @@ public static Destination sshWrappedDestination() {
 
   @Override
   public AirbyteConnectionStatus check(JsonNode config) {
-    var redisConfig = new RedisConfig(config);
-
-    RedisCache redisCache = null;
-    try {
-      redisCache = RedisCacheFactory.newInstance(redisConfig);
+    try (RedisCache redisCache = RedisCacheFactory.newInstance(config)) {
       // check connection and write permissions
       redisCache.ping("Connection check");
       return new AirbyteConnectionStatus().withStatus(AirbyteConnectionStatus.Status.SUCCEEDED);
     } catch (Exception e) {
       LOGGER.error("Can't establish Redis connection with reason: ", e);
-      return new AirbyteConnectionStatus().withStatus(AirbyteConnectionStatus.Status.FAILED);
-    } finally {
-      if (redisCache != null) {
-        redisCache.close();
-      }
+      return new AirbyteConnectionStatus()
```

Comment:
Could we add in an `AirbyteTraceMessage` before returning similar to this snippet so we have logging of the Exception?

https://github.com/airbytehq/airbyte/blob/84c7ff9cf50cf653bcedf4b186cbc09022eb6d67/airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/ssh/SshWrappedSource.java#L45-L46

Reply:
Yeah, did not notice that we have a standard for emitting config errors. Thanks! I'll update